content/200-concepts/100-components/02-prisma-client/056-composite-types.mdx

```diff
@@ -94,12 +94,12 @@ Conditions:
 
 - The composite type is [required](/concepts/components/prisma-schema/data-model#optional-and-mandatory-fields), and
 - the composite type has a [default value](/concepts/components/prisma-schema/data-model#defining-a-default-value), and
-- the database read operation returns the value `undefined`.
+- the composite type is not present in the returned document or documents.
```

Comment:
I'm just seeing this, but it's not the composite type that has a default or is really interesting here, it's a _field on the composite_. Consider:
```
models Test {
  // ...
  composite Composite?
}

type Composite {
  field String @default("test")
}
```

The composiste is optional, but that doesn't matter. What matters is that the field `field` on the composite is required and has a default. The "composite is x" is wrong in multiple places here.

Sorry for not catching that earlier.

Reply:
Thanks, I've fixed this now